lib/astutils.cpp

```diff
@@ -238,6 +238,16 @@ bool isSameExpression(bool cpp, bool macro, const Token *tok1, const Token *tok2
     return commutativeEquals;
 }
 
+bool equalTokValue(const Token * const tok1, const Token * const tok2)
```

Comment:
how about the name `isEqualKnownValue`.

I am a bit skeptic that this is non-static as it is only used in this file.

Reply:
I plan to use this for duplicateTernaryExpression check to improve it.